.github/workflows/release.yml

```
@@ -47,14 +47,18 @@ jobs:
           git config --local user.name "github-actions[bot]"
           git add docs
           git commit -m "Docs"
-      - name: Generate index
-        run: |
-          cd docs
-          tree -H '.' -L 3 --noreport --charset utf-8 -o index.html
+
       - name: Publish to pages
         uses: ad-m/github-push-action@master
         with:
           github_token: ${{ secrets.GITHUB_TOKEN }}
           branch: gh-pages
           directory: build
           force: true
+      - name: Qickgui
```

```suggestion
      - name: Quickgui
```
I spotted a small typo here :)